### PR TITLE
[Do Not Merge] Adding UserIdentities 

### PR DESCRIPTION
**Pull request by shail2208**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ y] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ y] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ y] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/machineLearningServices.json [hunk 1]
```diff
@@ -1905,12 +1905,34 @@
           "type": "string",
           "description": "The identity type.",
           "enum": [
-            "SystemAssigned"
+            "SystemAssigned",
+            "UserAssigned",
+            "SystemAssigned, UserAssigned",
+            "None"
```
Comment (weidongxu-microsoft) [inline specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/machineLearningServices.json]: Change enum with modelAsString=false is likely a breaking change.

Comment (shail2208) [reply specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/machineLearningServices.json]: @weidongxu-microsoft  Can you point us to how to add this extra identity types without breaking?

Comment (weidongxu-microsoft) [reply specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/machineLearningServices.json]: I am checking with other swagger review team members on this. I think there is no way to make it nonbreaking, given the previous definition.

I see it is 2020-01, so question is if you have SDK released or not. If not, you might be safe to modify.

Comment (shail2208) [reply specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/machineLearningServices.json]: I can do one thing. I will release a new version. "2020-02-01". I think that should fix the issue.

**Comment by weidongxu-microsoft** [comment]
@shail2208 

If you no longer need this PR, I will close it.

**Pull request closed without merging**